src/Sarif/ConsoleLogger.cs

```diff
@@ -263,12 +264,12 @@ private string FormatNotificationMessage(Notification notification)
 
             if (!string.IsNullOrEmpty(notification.Id))
             {
-                sb.Append($" {notification.Id}: ");
+                sb.AppendFormat(CultureInfo.CurrentCulture, "$" +notification.Id+": ");
             }
 
             if (!string.IsNullOrEmpty(notification.RuleId))
             {
-                sb.Append($"{notification.RuleId}: ");
+                sb.AppendFormat(CultureInfo.CurrentCulture, "$" + notification.RuleId + ": ");
```

Comment:
You need to get rid of the appended "$" here. It used to broadcast an interpolated string. Or you need to reintroduce the interpolated string literal as the argument to the call (if permitted, I assumed you made a change due to a compilation issue??)

https://msdn.microsoft.com/en-us/library/dn961160.aspx